src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -126,6 +126,15 @@ Integer linkLetterHolder(
         @Bind("letterHolderId") final String letterHolderId
     );
 
+    @SqlUpdate(
+        "UPDATE claim SET sealed_claim_document_management_self_path = :sealedClaimDocumentManagementSelfPath"
+            + " WHERE id = :claimId"
+    )
```

Comment:
`linkSealedClaimDocument`?

Reply:
Damiand commented to write just link as parameters talks about in details

Reply:
Ok, that's fine.

Reply:
@dharmendrak I meant renaming argument to be something generic e.g. `documentSelfPath`, not renaming method. I agree with @rlewan comment about having method named `linkSealedClaimDocument`, just do not use `sealedClaimDocumentManagementSelfPath` as argument name which is quite long.